core/domain/story_services_test.py

```diff
@@ -167,9 +167,8 @@ def test_update_story_properties(self):
             os.path.join(feconf.TESTS_DATA_DIR, 'test_svg.svg'),
             'rb', encoding=None) as f:
             raw_image = f.read()
-        fs = fs_domain.AbstractFileSystem(
-            fs_domain.GcsFileSystem(
-                feconf.ENTITY_TYPE_STORY, self.STORY_ID))
+        fs = fs_services.GcsFileSystem(
+            feconf.ENTITY_TYPE_STORY, self.STORY_ID)
```

Comment:
This can be moved to the previous line, right? ditto below

Reply:
Done.